lib/ex_doc.ex

```diff
@@ -21,12 +21,15 @@ defmodule ExDoc do
   Generates documentation for the given `project`, `version`
   and `options`.
   """
+  @spec generate_docs(String.t, String.t, Keyword.t) :: atom
   def generate_docs(project, version, options) when is_binary(project) and is_binary(version) and is_list(options) do
     config = build_config(project, version, options)
     docs = config.retriever.docs_from_dir(config.source_beam, config)
     find_formatter(config.formatter).run(docs, config)
   end
 
+  # Builds configuration by merging `options`, and normalizing the options.
+  # @spec build_config(String.t, String.t, Keyword.t) :: %ExDoc.Config{Keyword.t}
```

Comment:
%ExDoc.Config{Keyword.t} is not valid,
I just wanted you to review the specs if you could